### ISSUE TITLE
USER STORY: Hyperparameter Optimisation

### ISSUE THREAD

**Issue by Hasmelash95** [Hasmelash95/cherry-leaves-mildew-detection#12]
As a **Developer** I can **adjust hyperparameters** so that **the model's prediction accuracy increases and meets the business requirements**.

**Acceptance Criteria**

* Adjust the hyperparameters using the validation set to get the accuracy to above 97%.
* Plot the changes in accuracy and loss for training and validation sets to ensure normal fit.
